fluster/decoders/gstreamer.py

```diff
@@ -200,6 +199,34 @@ def gen_pipeline(
         )
 
 
+class GStreamerMpeg2v(GStreamer10Video):
+    """Base class for GStreamer mpeg2 video decoder"""
```

Comment:
As explained for the FFmpeg decoders, there is no need to specialize a decoder implementation for a pixel-comparison method since the current `decode` already allows decoding to an output file and keeping that file. There is no need for this class.